app/src/main/res/values/strings.xml

```diff
@@ -785,6 +785,12 @@ Before uploading your changes, the app checks with a &lt;a href=\"https://www.we
     <string name="quest_bicycle_rental_type_human">Staffed bicycle rental</string>
     <string name="quest_bicycle_rental_type_shop_with_rental">Bicycle store that also rents bicycles</string>
 
+    <string name="quest_bicycle_repair_station_services_title">What equipment is available here?</string>
+    <string name="quest_bicycle_repair_station_pump">Pump</string>
```

Comment:
```suggestion
    <string name="quest_bicycle_repair_station_pump">Pump (operational)</string>
```

or "working pump", "functional pump" or similar wording. Rationale why that emphasis is needed in previous discussion: https://github.com/streetcomplete/StreetComplete/issues/3053#issuecomment-879326785

Reply:
I was thinking about this, too, but I decided against this because:
1. wiki gives no indication whether the tag is about a physical presence of a pump or that it is working
2. other than for obvious cases, e.g. where the tube has been torn off, it might not be easily possible for the mapper to notice when a pump doesn't work. If the pump is damaged enough that pieces are missing, I'd expect that mappers either answer "no" or leave a note also without any change in wording.